datanode/gateway/graphql/schema.graphql

```diff
@@ -1899,10 +1891,10 @@ type PriceLevel {
 "Candle stick representation of trading"
 type Candle {
   "Unix epoch+nanoseconds for when the candle occurred"
-  timestamp: String!
+  periodStart: String!
 
   "RFC3339Nano formatted date and time for the candle"
-  datetime: String!
+  lastUpdateInPeriod: String!
```

Comment:
Should the doc comments be updated?

Reply:
Sorry, always forget about the Docstrings. I will address.